virtual-lab-manager/elasticache.tf

```
@@ -0,0 +1,37 @@
+
+
+
+resource "aws_security_group" "vlm_redis_sg" {
+  name        = "vlm-redis-sg"
+  description = "Security group for Redis instance for Virtual Lab Manager"
+
+  vpc_id = var.vpc_id
+
+  ingress {
+    from_port   = 6379
+    to_port     = 6379
+    protocol    = "tcp"
+    cidr_blocks = [var.vpc_cidr_block]
```

The subnet itself is a private one, meaning there is already no access from outside of VPC.
I would recommend to put a wildcard here for now, so there is no confusion: removing the value here doesn't make Redis publicly accessible.

---

do you mean ports to be 0-0 ?

---

For the ports, the current values are actually good security wise.

For the cidr_blocks:
```suggestion
    cidr_blocks = ["0.0.0.0/0"]
```